### DISCUSSION TITLE
Theme is rendering in very very strange way

### DISCUSSION THREAD

**Discussion by sudipghimire533** [uPagge/uBlogger#108] [Q&A]
Ok I just followed the guide and copied one of the post from exampleSite of this repo. And below is the rendeing I have got
![image](https://user-images.githubusercontent.com/39845901/120777625-33f45d00-c545-11eb-915e-800f67b2a155.png)
However other sites including [uBlogger home page](https://ublogger.netlify.app/) are working just perfect. I can't even figure out what is wrong here.

**Comment by uPagge** [comment]
it would be nice if you did a demo project

**Reply by sudipghimire533** [reply]
Hello there again.
I just tried making a samlpe content just to test this. And this is still what I got. Strange but yes this is what I got.

![image](https://user-images.githubusercontent.com/39845901/120899139-e9123c80-c64d-11eb-9dad-0f4738553fc4.png)

I even dont have any hints what is causing this.
